orb/src/jobs/e2e-test-review.yml

```diff
@@ -10,3 +10,5 @@ steps:
   - run:
       name: Smoke tests
       command: npx dotcom-tool-kit test:review
+  - store_artifacts:
+      path: /e2e-test-results
```

Comment:
**suggestion**: we should use relative paths from the build directory instead of storing artefacts at the root of the filesystem (I'm not sure we even have permission to create new directories at root?)
```suggestion
      path: e2e-test-results
```

Reply:
this isn't a path to create artefacts in, it's the (existing) path to save files from as artefacts

Reply:
I'm assuming we'll have to configure Cypress (and any other e2e tester) to store artefacts in this `e2e-test-results` directory and I don't think we should be configuring it to store the results at `/e2e-test-results`

Reply:
oh yeah I already store artifacts in cypress/videos so will just set to that

Reply:
because these orb tasks could be used for another e2e test plugin not just Cypress, the path it's saving should be generic, and we should configure Cypress to save videos to that generic path